src/style.css

```
@@ -0,0 +1,125 @@
+* {
+  outline: none;
+  box-sizing: border-box;
+}
+
+@font-face {
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  font-weight: 300;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
+}
+
+.big__bar {
+  margin: 20px 0;
+}
+
+.search__bar {
+  width: 100%;
+  height: 70px;
+  flex-shrink: 0;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+}
+
+.icon--big {
+  margin: 25px 17px 26px 26px;
+  width: 19px;
+  height: 19px;
+  flex-shrink: 0;
+  color: #687c94;
+}
+
+.biginput {
+  outline: none;
+  border: none;
+  color: #3d4e61;
+  /* padding-top: 25px; */
+  display: flex;
+  align-items: center;
+  align-content: center;
+}
+
+.search__input {
+  outline: none;
+  padding: 0;
+  border: none;
+  font-family: "Avenir", sans-serif;
+  color: #3d4e61;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
```

Default value, so you can just remove it everywhere
```suggestion
  font-weight: 300;
```

---

Not resolved

---

Я переробив код з того разу. Зараз ці значення не за замовчкванням, і якщо їх видаляти то все зламається.